lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.install

```diff
@@ -90,3 +92,20 @@ function dosomething_gladiator_update_7002(&$sandbox) {
     }
   }
 }
+
+/**
+ * Update response values to be a long text field, as the response is LONG.
```

Comment:
lol just making sure, the title of the PR says you were updating to medium text, but this says long?